### PR TITLE
fix: redirect to table page after saving the edit

### PR DESCRIPTION
**Pull request by pazolka**
and show fields with multiple values in the table entry view

Use with library branch https://github.com/unepwcmc/wcmc-components/pull/66/files

### CODE REVIEW

File: wcmc_components/app/controllers/wcmc_components/table_controller.rb [hunk 1]
```diff
@@ -48,7 +49,7 @@ def update
 
       if @table_resource.save
         # Redirect (to the show page/index page)
-        render :edit
```
Comment (sergiomarrocoli) [inline wcmc_components/app/controllers/wcmc_components/table_controller.rb:51]: also needs to be added to new and edit actions, and for now including in the create links in the IR pages (should eventually be a button in the front end library)

**Review by sergiomarrocoli — CHANGES_REQUESTED** [review]

**Comment by lucacug** [comment]
@sergiomarrocoli  can you have a look at this, please? Also, not sure why this is pointing to the `fix/show-page-multiple-attributes' branch, can this be merged into develop?

File: README.md [hunk 2]
```diff
@@ -266,6 +266,11 @@ else
 end
 ```
 
+The redirect after the new/edit is set to default to the index path of the created/updated resource, e.g.
```
Comment (sergiomarrocoli) [inline README.md:269]: could this be a subsection in ### configure the engine? or have a heading of some kind to make it a bit easier to find?

**Review by sergiomarrocoli — CHANGES_REQUESTED** [review]
works perfectly, one comment about the readme

File: README.md (hunk 2 above)
Comment (lucacug) [reply README.md]: Heading added

**Review by sergiomarrocoli — APPROVED** [review]

**Pull request merged**